roles/setup_gitea/templates/role_scc_anyuid.j2

```diff
@@ -0,0 +1,14 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: scc-anyuid
+  namespace: "{{ sg_namespace }}"
+rules:
+  - apiGroups:
+      - security.openshift.io
+    resourceNames:
+      - anyuid
+    resources:
+      - securitycontextconstraints
+    verbs:
+      - use
```

Comment:
Missed line break at the end of the file

Reply:
Done